entry-bgp.sh

```diff
@@ -122,14 +122,25 @@ EOF
 		done
 	fi
 
+	if [ -n "$BGP_IPV6" -o "$BGP_IPV6" = yes ]; then
```

Comment:
The condition evaluates to true for BGP_IPV6=no (which is probably not intended). 
Also `-n $x` is always true when `$x = foo` this "or" makes no sense.